lbry/extras/daemon/daemon.py

```diff
@@ -3712,7 +3712,7 @@ async def jsonrpc_collection_create(
 
         Usage:
             collection_create (<name> | --name=<name>) (<bid> | --bid=<bid>)
-                    (--claims=<claims>...)
+                    (--claims=<claims>... | --claims=<claims>)
```

Comment:
what does this change do?

Reply:
thought that was necessary for cli / docs generation. guess not.